readme.md

```diff
@@ -40,9 +40,10 @@ Pact.crypto.toTweetNaclSecretKey(keyPair) -> <Uint8Array>
 ### Language Expression Construction
 
 A helper function for constructing native Pact commands.
-
+- mkExp takes in Pact function and its arguments and returns a Pact expression.
+- mkMeta returns an object of metadata of the tx. "sender" represents the gas account, "ChainId" represents the Chain Id, "gasPrice" represents the price of gas, and "gasLimit" represents the limit of gas.
```

Comment:
Note also that:
- this is obviously only important for public blockchain contexts, private blockchain doesn't need gas
- In the public blockchain context, it's very important that SENDER also signs the transaction with the keyset associated with the account, otherwise the transaction will get rejected.

Reply:
Edited the mkMeta description.